examples/v1/pipelineruns/beta/http-resolver.yaml

```diff
@@ -11,7 +11,7 @@ spec:
           resolver: http
           params:
             - name: url
-              value: https://api.hub.tekton.dev/v1/resource/tekton/task/tkn/0.4/raw
+              value: https://api.hub.tekton.dev/v1/resource/tekton/task/git-clone/0.10/raw
         params:
-          - name: ARGS
-            value: ["version"]
+          - name: url
```

Comment:
We need to update pipelinerun with workspace if we really want to go with git-clone here.